tests/README.md

```diff
@@ -0,0 +1,22 @@
+# SEBAK Integration tests
+
```

Comment:
We should list the testsuite dependencies here (`jq`) as @outsideris pointed out